packaging/version.py

```diff
@@ -90,14 +90,42 @@ def base_version(self):
         return self._version
 
     @property
+    def epoch(self):
+        return -1
```

Comment:
To match the version comparison code's treatment of `LegacyVersion`s as having an epoch of -1, reflecting the fact that `LegacyVersion`s always sort less than `Version`s (cf. https://git.io/vQoHZ)